models/votes.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (sequelize, DataTypes) {
+  const Vote = sequelize.define("Vote", {
+    voteId: {type: DataTypes.INTEGER, allowNull: false},
+    quoteId: {type: DataTypes.INTEGER, allowNull: false},
+    categoryId: {type: DataTypes.INTEGER, allowNull: false},
+    numberOfVotes: {type: DataTypes.INTEGER, allowNull: false} });
+
+  //Vote.associate = function (models) {
+    //Vote.belongsTo(models.Votes, {
+     // foreignKey: { allowNull: false}
+   // });
+  };
```

Comment:
You have an extra bracket and here on line 12 and semicolon. You already closed the object in the function on line 6.